website/docs/d/users.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "gitlab"
+page_title: "GitLab: gitlab_users"
+sidebar_current: "docs-gitlab-data-source-users"
+description: |-
+  Looks up gitlab users
+---
+
+# gitlab\_users
+
+Provides details about a list of users in the gitlab provider. The results include id, username, email, name and more about the requested users. Users can also be sorted and filtered using several options.
+
+## Example Usage
+
+```hcl
+data "gitlab_users" "example" {
+  sort = "desc"
+  order_by = "name"
+  created_before = "2019-01-01"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `search` - (Optional) Search users by username, name or email.
+
+* `active` - (Optional) Filter users that are active.
+
+* `blocked` - (Optional) Filter users that are blocked.
+
+* `order_by` - (Optional) Order the users' list by `id`, `name`, `username`, `created_at` or `updated_at`. (Requires administrator privileges)
```

Comment:
A top level note should be added to explain why this requires administrator privileges.

Is sorting by `name` or `username` also requiring those privileges?

Reply:
Does a link to the doc would be enough? There's no really any explanation about the why this is required except that it's in the `For admins` section of the doc (cf. https://docs.gitlab.com/ce/api/users.html#for-admins)

Reply:
A link is great!

Reply:
Done.